src/quo/components/links/internal_link_card/channel/view.cljs

```diff
@@ -0,0 +1,89 @@
+(ns quo.components.links.internal-link-card.channel.view
+  (:require
+    [quo.components.icon :as icon]
+    [quo.components.links.internal-link-card.channel.style :as style]
+    [quo.components.markdown.text :as text]
+    [quo.theme :as quo.theme]
+    [react-native.core :as rn]))
+
+(defn- description-comp
+  [description]
+  [rn/view {:style {:margin-bottom 4}}
+   [text/text
+    {:size                :paragraph-2
+     :number-of-lines     3
+     :accessibility-label :description}
+    description]])
+
+(defn- title-comp
+  [title channel-name theme]
+  [rn/view
+   {:style {:flex-direction :row
+            :align-items    :center}}
+   [text/text
+    {:size                :paragraph-1
+     :number-of-lines     1
+     :weight              :semi-bold
+     :style               (dissoc style/title :flex)
+     :accessibility-label :title}
+    title]
+   [icon/icon :i/chevron-right (style/channel-chevron-props theme)]
+   [text/text
+    {:size                :paragraph-1
+     :number-of-lines     1
+     :weight              :semi-bold
+     :style               (dissoc style/title :flex)
+     :accessibility-label :title}
+    channel-name]])
+
+(defn- banner-comp
+  [thumbnail]
+  [rn/image
+   {:style               style/thumbnail
+    :source              thumbnail
+    :accessibility-label :banner}])
+
+(defn- logo-comp
+  [logo]
+  [rn/image
+   {:accessibility-label :logo
+    :source              logo
+    :style               (assoc style/logo :margin-bottom 2)}])
+
+(defn- loading-view
+  [theme]
+  [rn/view
+   {:accessibility-label :loading
+    :style               {:height 215}}
+   [rn/view {:style {:flex-direction :row}}
+    [rn/view {:style style/row-spacing}
+     [rn/view (style/loading-circle theme)]
+     [rn/view (assoc (style/loading-first-line-bar theme) :margin-right 4)]]
```

Comment:
for this single-assoc why not just make that style function take a param of `margin-right?`. It's much easier to see what is going on with that style then.